var/graphs/mbta/router-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "stopClusterMode": "parentStation",
+  "useFlexService": "true",
+  "updaters": [
+    {
+      "type": "real-time-alerts-enhanced",
+      "frequencySec": 100,
+      "url": "https://cdn.mbta.com/realtime/Alerts_enhanced.json",
+      "feedId": "1"
+    },
+    {
+      "type": "stop-time-updater",
+      "frequencySec": 1000,
```

Comment:
Is this updated every 16 minutes?

Reply:
Same as above, should be 10 seconds, fixed.